server/src/schema.js

```diff
@@ -5,10 +5,14 @@ const typeDefs = gql`
         name: String!
         price: Int!
         imgUrl: String!
+        selected: Int!
     }
     type Query {
         getItems: [Item]!
     }
+    type Mutation {
+        updateSelected(selected: [Int!]!): Boolean!
```

Comment:
`[Int]!` seems a more accurate form since there could be a case that has 0 items, don't you think?